spec/bolt/module_installer_spec.rb

```diff
@@ -54,6 +54,17 @@
       )
     end
 
+    it 'errors if module is already in puppetfile without version' do
+      File.write(puppetfile, new_module)
```

Comment:
```suggestion
      File.write(puppetfile, "mod '#{new_module}'")
```

We want to write a valid Puppetfile instead of writing just the module name. [This](https://puppet.com/docs/pe/latest/puppetfile.html) doc goes over the Puppetfile format.